src/components/calcite-input/calcite-input.e2e.ts

```diff
@@ -311,7 +311,7 @@ describe("calcite-input", () => {
     expect(await element.getProperty("value")).toBe("12");
     expect(await element.getProperty("min")).toBe(10);
     expect(await element.getProperty("max")).toBe(15);
-    expect(await element.getProperty("step")).toBe("1");
```

Comment:
This was kind of weird to be a string. If min/max are numbers why can't step be?

Reply:
Yeah, if its checking the property, it should be a number, not a string.  Only the attribute would be set to a string.